new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java

```diff
@@ -0,0 +1,309 @@
+    for (int i = 1; i <= replicaCount; i++) {
+      final String managedServerPodName = managedServerPrefix + i;
+      managedServerPodOriginalTimestampList.add(
+          assertDoesNotThrow(() -> getPodCreationTimestamp(domainNamespace, "", managedServerPodName),
+              String.format("getPodCreationTimestamp failed with ApiException for pod %s in namespace %s",
+                  managedServerPodName, domainNamespace)));
+    }
+
+    // create ingress for the domain
+    logger.info("Creating an ingress for the domain");
+    createIngressForDomain();
+
+    // verify the sample apps for the domain
+    logger.info("Checking that the sample app can be accessed from all managed servers through NGINX");
+    verifySampleAppAccessThroughNginx();
+
+    // delete operator
+    logger.info("Uninstalling operator");
+    uninstallOperator(opHelmParams);
+
+    // verify the operator pod was deleted
+    logger.info("Checking that operator pod was deleted");
+    checkPodDeleted("weblogic-operator-", null, opNamespace);
+
+    // verify the operator service was deleted
+    logger.info("Checking that operator service was deleted");
+    checkServiceDeleted(OPERATOR_SERVICE_NAME, opNamespace);
+
+    // check that the state of admin server pod in the domain was not changed
+    logger.info("Checking that the admin server pod state was not changed after the operator was deleted");
+    assertThat(podStateNotChanged(adminServerPodName, domainUid, domainNamespace, adminPodOriginalTimestamp))
+        .as("Test state of pod {0} was not changed in namespace {1}", adminServerPodName, domainNamespace)
+        .withFailMessage("State of pod {0} was changed in namespace {1}",
+            adminServerPodName, domainNamespace)
+        .isTrue();
+
+    // check that the states of managed server pods in the domain were not changed
+    logger.info("Checking that the managed server pod state was not changed after the operator was deleted");
+    for (int i = 1; i <= replicaCount; i++) {
+      String managedServerPodName = managedServerPrefix + i;
+      assertThat(podStateNotChanged(managedServerPodName, domainUid, domainNamespace,
+              managedServerPodOriginalTimestampList.get(i - 1)))
+          .as("Test state of pod {0} was not changed in namespace {1}", managedServerPodName, domainNamespace)
+          .withFailMessage("State of pod {0} was changed in namespace {1}",
+              managedServerPodName, domainNamespace)
+          .isTrue();
+    }
+
+    // verify the sample app in the domain is still accessible from all managed servers through NGINX
+    logger.info("Checking that the sample app can be accessed from all managed servers through NGINX "
+        + "after the operator was deleted");
+    verifySampleAppAccessThroughNginx();
+  }
+
+  /**
+   * TODO: remove this after Sankar's PR is merged
+   * The cleanup framework does not uninstall NGINX release. Do it here for now.
```

Comment:
Check with Sankar. I think his PR has been mergrd in develop

Reply:
not yet.